_data/projects/ihatemoney.yml

```diff
@@ -0,0 +1,11 @@
+name: I Hate Money
+desc: A simple shared budget manager web application
+site: https://github.com/spiral-project/ihatemoney
+tags:
+- python
+- flask
+- budget
+- rest
+upforgrabs:
+  name: good first issue
+  link: https://github.com/spiral-project/ihatemoney/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22
```

Comment:
@almet Please change this to https://github.com/spiral-project/ihatemoney/labels/good%20first%20issue

Reply:
Here we go!